### DISCUSSION TITLE
Last.fm integration - or a similar service

### DISCUSSION THREAD

**Discussion by isitreallyalive** [ppy/osu#16202] [Ideas] [answered]
I have no clue on the logistics of something like this, but do you reckon it would be possible to have a way to scrobble songs played on osu! to Last.fm? (or any similar service) You could perhaps connect your account on the website or in game, and then before or after score submission have the song scrobbled to your account. I think this would be a pretty neat feature to have!

**Comment by bdach** [comment, marked as answer]
last.fm was a thing in stable already but got excised due to generally being behind the times so to speak (https://github.com/ppy/osu/pull/10518). So I wouldn't hold my breath for that.

Not sure if there are any other similar services it would be worth spending the time for.
